import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -44,19 +52,25 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
       credentials = credentials.createScoped(Arrays.asList(flags.oauthScopes.split(",")));
     }
 
+    HttpRequestFactory requestFactory =
+        new NetHttpTransport().createRequestFactory(new HttpCredentialsAdapter(credentials));
+
+    // Initialize Monitoring
+    MonitoringService.initialize(flags.monitoringProjectId, Event.values(), requestFactory);
```

Comment:
Ok

Reply:
As a consequence, now monitoringProjectId became a mandatory parameter to enable monitoring (while before it could be automatically taken from kubernetes environment).